src/components/measureLanding/measureList/createVersionDialog/CreateVersionDialog.tsx

```diff
@@ -97,107 +93,130 @@ const CreatVersionDialog = ({
     },
   });
 
-  const classes = useStyles();
-  const flexEnd = classNames(classes.row, classes.end);
+  const determineVersionNumber = useCallback(() => {
```

Comment:
I thought useCallback is only helpful in stopping unnecessary renders for child components when its props didn't change. Do you think we still need it here ?
My thinking is that even if just use useEffect and call getNewVersion, unless we set a value to setNewVersionNumber, re-render doesn't happen.

Reply:
I believe so. Updated accordingly